src/modules/auth/decorators/get-user.decorator.ts

```diff
@@ -5,11 +5,13 @@ import {
 } from '@nestjs/common';
 
 export const GetUser = createParamDecorator(
-  (data: unknown, ctx: ExecutionContext) => {
+  (data: string | undefined, ctx: ExecutionContext) => {
```

Comment:
Let's type it, what fields can be there

Reply:
You can use `keyof` with a type for `request.user` if there is any.